main/exercise/question.class.php

```diff
@@ -1911,22 +2056,72 @@ public static function displayTypeMenu($objExercise)
         echo '</div>';
     }
 
+    /**
+     * Displays the menu of question types.
+     *
+     * @param Exercise $objExercise
+     */
+    public static function displayPtTypeMenu($objExercise)
+    {
+        // 1. by default we show all the question types
+        $questionTypeList = self::getQuestionPtTypeList();
+
+        echo '<div class="panel panel-default">';
+        echo '<div class="panel-body">';
+        echo '<ul class="question_menu">';
+        foreach ($questionTypeList as $i => $type) {
+            /** @var Question $type */
+            $type = new $type[1]();
+            $img = $type->getTypePicture();
+            $explanation = get_lang($type->getExplanation());
+            echo '<li>';
+            echo '<div class="icon-image">';
+            $icon = '<a href="ptest_admin.php?'.http_build_query([
+                'exerciseId' => $objExercise->id,
+                'newQuestion' => 'yes',
+                'answerType' => $i,
+            ]).'&'.api_get_cidreq().'">'.Display::return_icon($img, $explanation, null, ICON_SIZE_BIG).'</a>';
+
+            if ($objExercise->force_edit_exercise_in_lp === false) {
+                if ($objExercise->exercise_was_added_in_lp == true) {
+                    $img = pathinfo($img);
+                    $img = $img['filename'].'_na.'.$img['extension'];
+                    $icon = Display::return_icon($img, $explanation, null, ICON_SIZE_BIG);
+                }
+            }
+            echo $icon;
+            echo '</div>';
+            echo '</li>';
+        }
+        echo '<li class="pull-right">';
+        echo '<div class="icon-image">';
+        $icon = '<a href="ptests_category.php?exerciseId='.$objExercise->selectId().'&'.api_get_cidreq().'">'.
+            Display::return_icon('sessions_category.png', get_lang('PtCategories'), null, ICON_SIZE_BIG).
+            '</a>';
+        echo $icon;
+        echo '</div>';
+        echo '</li>';
+        echo '</ul>';
+        echo '</div>';
+        echo '</div>';
+    }
+
     /**
      * @param int    $question_id
      * @param string $name
-     * @param int    $course_id
+     * @param int    $courseId
      * @param int    $position
      *
      * @return false|string
      */
-    public static function saveQuestionOption($question_id, $name, $course_id, $position = 0)
+    public static function saveQuestionOption($question_id, $name, $courseId, $position = 0)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The variable $last_id is not named in camelCase.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)